gtsfm/utils/metrics.py

```diff
@@ -53,6 +53,8 @@ def count_correct_correspondences(
 
     Returns:
         Number of correspondences which are correct.
+        Mask of which verified correspondences are classified as correct under Sampson error
+            (using GT epipolar geometry).
```

Comment:
I dont think this is a good API, its like returning a mask and a non-zero count of the mask. The user could just count the nonzeros outside.